src/analysisd/decoders/syscollector.c

```diff
@@ -29,16 +29,162 @@ static int prev_port_id = 0;
 static int error_process = 0;
 static int prev_process_id = 0;
 
-static int decode_netinfo( Eventinfo *lf, cJSON * logJSON,int *socket);
-static int decode_osinfo( Eventinfo *lf, cJSON * logJSON,int *socket);
-static int decode_hardware( Eventinfo *lf, cJSON * logJSON,int *socket);
-static int decode_package( Eventinfo *lf, cJSON * logJSON,int *socket);
+static int decode_netinfo( Eventinfo *lf, cJSON * logJSON, int *socket);
+static int decode_osinfo( Eventinfo *lf, cJSON * logJSON, int *socket);
+static int decode_hardware( Eventinfo *lf, cJSON * logJSON, int *socket);
+static int decode_package( Eventinfo *lf, cJSON * logJSON, int *socket);
 static int decode_hotfix(Eventinfo *lf, cJSON * logJSON, int *socket);
-static int decode_port( Eventinfo *lf, cJSON * logJSON,int *socket);
-static int decode_process( Eventinfo *lf, cJSON * logJSON,int *socket);
+static int decode_port( Eventinfo *lf, cJSON * logJSON, int *socket);
+static int decode_process( Eventinfo *lf, cJSON * logJSON, int *socket);
+static int decode_dbsync( char const *agent_id, char *msg_type, cJSON * logJSON, int *socket);
 
 static OSDecoderInfo *sysc_decoder = NULL;
 
+static const char* hotfixes_fields[] = { 
```

Comment:
Please, change this static const to ALL_CAPS_WITH_UNDERSCORES to be compliant with the Coding Style Guide (C)